flink-tests/src/test/scala/org/apache/flink/api/scala/migration/StatefulJobSavepointMigrationITCase.scala

```diff
@@ -159,7 +152,7 @@ class StatefulJobSavepointMigrationITCase(
     restoreAndExecute(
       env,
       SavepointMigrationTestBase.getResourceFilename(
-        s"stateful-scala${StatefulJobSavepointMigrationITCase.SCALA_VERSION}" +
```

Comment:
So the state is not dependent on the scala version at all then? i.e. i can load a scala 2.11 snapshot in 2.12, and vice-versa? (if not I'd keep the version suffix to make things more explicit)